bc_obps/.coveragerc

```diff
@@ -0,0 +1,11 @@
+[run]
+omit =
+  *apps.py,
+  */migrations/*,
+  *settings*,
+  */tests/*,
```

Comment:
I don't fully understand what this file is doing, but seems odd to omit the test directory?

Reply:
This file serves as the configuration file for the pytest coverage package. Its purpose is to define the folders that should not be checked against the tests in the `tests` folder. Essentially, this means that we want to avoid testing the test code itself. Additionally, we also want to exclude files such as migration files and settings from being tested.